extensions/amp-story/1.0/utils.js

```diff
@@ -211,33 +205,13 @@ export function userAssertValidProtocol(element, url) {
  * @return {string}
  */
 export function getSourceOriginForElement(element, url) {
-  let domainName;
-
+  const urlService = Services.urlForDoc(element);
   try {
-    domainName = getSourceOrigin(Services.urlForDoc(element).parse(url));
-    // Remove protocol prefix.
-    domainName = Services.urlForDoc(element).parse(domainName).hostname;
-  } catch (e) {
+    url = urlService.getSourceOrigin(url);
+  } catch (_) {
     // Unknown path prefix in url.
-    domainName = Services.urlForDoc(element).parse(url).hostname;
-  }
-  return domainName;
-}
-
-/**
- * Resolves an image url and optimizes it if served from the cache.
- * @param {!Window} win
- * @param {string} url
- * @return {string}
- */
-export function resolveImgSrc(win, url) {
```

Comment:
This function is [unused](https://github.com/ampproject/amphtml/search?q=resolveImgSrc)